### PR TITLE
Merge `noetic-devel`

### PR DESCRIPTION
**Pull request by PaperFanz**
Fixed properties pane not updating on filter drag.

Fixed `qt_build` dependency.

Updated documentation (ongoing).

### CODE REVIEW

File: insitu/scripts/template/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## System dependencies are found with CMake's conventions
+find_package(Qt5 COMPONENTS Core Gui Widgets)
+
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES {0}
+#  CATKIN_DEPENDS nodelet pluginlib roscpp
+#  DEPENDS system_lib
+)
```
Comment (AdamPettinger) [inline insitu/scripts/template/CMakeLists.txt]: The intent here is for users to uncomment as needed right? I like that, but maybe the `INCLUDE_DIRS` line should be uncommented automatically in the template?

File: qt_build/CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+^^^^^^^^^
+Changelog
+^^^^^^^^^
+
+0.2.7 (2014-07-27)
+------------------
+* update for indigo
```
Comment (AdamPettinger) [inline qt_build/CHANGELOG.rst:7]: Nice

**Review by AdamPettinger — APPROVED** [review]
Briefly looked through, looks great! Templating is awesome...

I remember talking to you about the `qt_build` stuff. Adding it here isn't very pretty but if that is what we gotta do... Let's do it and move on.

Eventually I want to install/play with InSitu on my laptop. The active development is on Noetic right?

**Comment by PaperFanz** [comment]
> I remember talking to you about the `qt_build` stuff. Adding it here isn't very pretty but if that is what we gotta do... Let's do it and move on.

Yeah, I can't think of a better way to do it ATM. Best is for `qt_build` to get packaged for Noetic.

> Eventually I want to install/play with InSitu on my laptop. The active development is on Noetic right?

Yes. Right now I'm still working to bring some features from `melodic-devel` over to `noetic-devel` but all future development will be focused on Noetic (and ROS2, eventually).

File: insitu/scripts/template/CMakeLists.txt (hunk 1 above)
Comment (PaperFanz) [reply insitu/scripts/template/CMakeLists.txt]: Good catch, pushed a new commit to fix. Needs testing on Noetic before I follow through with merge.

**Pull request merged**